### PR TITLE
add Resolve

### PR DESCRIPTION
**Pull request by Swiftkage**

### CODE REVIEW

**Comment by Swiftkage** [comment]
help @CS2103-AY1819S1-W12-1/developers i have no idea why the checks are failing

**Comment by dan-delion** [comment]
Make sure to pull changes from upstream before making a pr. It seems you're still using. `person`, which is no longer supported

File: src/main/java/seedu/address/logic/commands/ResolveCommand.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
```
Comment (dan-delion) [inline src/main/java/seedu/address/logic/commands/ResolveCommand.java]: This needs to be changed to article

Comment (dan-delion) [reply src/main/java/seedu/address/logic/commands/ResolveCommand.java]: ResolveCommand: Line 12, 15, 22, 26, 37, 40, 43, 44, 46

Comment (dan-delion) [reply src/main/java/seedu/address/logic/commands/ResolveCommand.java]: Please try testing the build in your machine before making a pr. See DeveloperGuide#Testing for instructions

Comment (Swiftkage) [reply src/main/java/seedu/address/logic/commands/ResolveCommand.java]: oops yea i forgot to get the upstream, so I was passing my own build on my machine

**Pull request merged**